ionic/__main__.py

```diff
@@ -788,6 +495,124 @@ async def on_lb_start(event: lb.LightbulbStartedEvent):
     )
 
 
+@bot.command()
+@lb.option("nojoy", "Ignores :rofl: and :joy: reactions",bool, default=True)
+@lb.option(
+    "hours",
+    "How far back should I pull messages from (int)",
+    int,
+    default=6,
+    max_value=24,
+    min_value=1,
+)
+@lb.option("channel", "Target channel to search", h.TextableGuildChannel, default=None)
```

Comment:
Would maybe switching around the order of these be better for the UX?

Having channel first seems like it might be better, happy to hear your opinions on this of course

Reply:
I have a similar opinion to yours, so I will swap the order of the options!